src/app/dashboard/component/details/details.component.html

```diff
@@ -2,231 +2,309 @@
+                    }
+
+                    @if (bioSampleObj.geographicLocationDepth?.text) {
+                        <ng-container>
+                            <dt class="col-sm-3">Geographic Location (Depth):</dt>
+                            <dd class="col-sm-9">{{bioSampleObj.geographicLocationDepth.text}}</dd>
+                        </ng-container>
+                    }
+
+                    @if (bioSampleObj.geographicLocationElevation?.text) {
+                        <ng-container>
+                            <dt class="col-sm-3">Geographic Location Elevation:</dt>
+                            <dd class="col-sm-9">{{bioSampleObj.geographicLocationElevation.text}}</dd>
+                        </ng-container>
+                    }
+
+                    @if (bioSampleObj.habitat) {
+                        <ng-container>
+                            <dt class="col-sm-3">Habitat:</dt>
+                            <dd class="col-sm-9">{{bioSampleObj.habitat}}</dd>
+                        </ng-container>
+                    }
+
+                    @if (bioSampleObj.identifiedBy) {
+                        <ng-container>
+                            <dt class="col-sm-3">Identified By:</dt>
+                            <dd class="col-sm-9">{{bioSampleObj.identifiedBy}}</dd>
+                        </ng-container>
+                    }
+
+                    @if (bioSampleObj.identifierAffiliation) {
+                        <ng-container>
+                            <dt class="col-sm-3">Identifier Affiliation:</dt>
+                            <dd class="col-sm-9">{{bioSampleObj.identifierAffiliation}}</dd>
+                        </ng-container>
+                    }
+
+                    @if (bioSampleObj.specimenVoucher) {
+                        <ng-container>
+                            <dt class="col-sm-3">Specimen Voucher:</dt>
+                            <dd class="col-sm-9">{{bioSampleObj.specimenVoucher}}</dd>
+                        </ng-container>
+                    }
+
+                    @if (bioSampleObj.projectName) {
+                        <ng-container>
+                            <dt class="col-sm-3">Project Name:</dt>
+                            <dd class="col-sm-9">{{bioSampleObj.projectName}}</dd>
+                        </ng-container>
+                    }
+
+                    @if (bioSampleObj.sampleDerivedFrom) {
+                        <ng-container>
+                            <dt class="col-sm-3">Sample Derived From:</dt>
+                            <dd class="col-sm-9">
+                                <a href="http://www.ebi.ac.uk/biosamples/samples/{{bioSampleObj.sampleDerivedFrom}}" target="_blank">
+                                    {{bioSampleObj.sampleDerivedFrom}}
+                                </a>
+                            </dd>
+                        </ng-container>
+                    }
+                </ng-container>
+            }
```

Comment:
You are not using "benefits" of new @if, etc... With @if there is no need to have a ng-container, so this code can be simplified to:
```
@if (bioSampleObj) {
                @if (bioSampleObj.commonName) {
                    <dt class="col-sm-3">Name:</dt>
                    <dd class="col-sm-9">{{bioSampleObj.commonName}}</dd>
                }
...
```

Reply:
also can you test if you need to have nested conditions here, can we just use
```
@if (bioSampleObj?.commonName)
```